README.md

```diff
@@ -39,19 +39,14 @@ With the ESnext field declarations proposal, the above example can be written as
 class Counter extends HTMLElement {
```

Comment:
Yep, that's still supported. But then I couldn't show up my @bound decorator in the larger decorators example (which is only visible in https://github.com/littledan/proposal-unified-class-features/blob/master/README.md ) :)! I'll switch to your coding pattern, as it's a frequently requested feature and possible with this proposal.